### ISSUE TITLE
Test suite fails with older pytorch version

### ISSUE THREAD

**Issue by gaurav274** [georgia-tech-db/eva#526]
```
E           RuntimeError: Failed to import MVITActionRecognition from eva/eva/udfs/mvit_action_recognition.py
E           Exception: cannot import name 'MViT_V2_S_Weights' from 'torchvision.models.video' (eva/env38/lib/python3.8/site-packages/torchvision/models/video/__init__.py). Please verify that the UDF class name in the implementation file matches the provided UDF name MVITActionRecognition.
```

**Comment by gaurav274** [comment]
Possible fix: request to install correct version of PyTorch if using `MVITActionRecognition`

**Comment by jarulraj** [comment]
Can we throw an error in the UDF based on PyTorch version similar to this -- 

`self.queue = asyncio.Queue() if sys.version_info >= (3, 10) else asyncio.Queue(loop=loop)`

**Comment by gaurav274** [comment]
Fixed with #530
